File: get-feature-265.js
const sqlite3 = require('sqlite3');

const db = new sqlite3.Database('./features.db');
db.get('SELECT * FROM features WHERE id = 265', (err, row) => {
  if (err) {
    console.error(err);
  } else {
    console.log(JSON.stringify(row, null, 2));
  }
  db.close();
});

File: get-feature-265.py
import sqlite3
import json

conn = sqlite3.connect('features.db')
cursor = conn.cursor()
cursor.execute('SELECT * FROM features WHERE id = 265')
row = cursor.fetchone()
if row:
    columns = [desc[0] for desc in cursor.description]
    print(json.dumps(dict(zip(columns, row)), indent=2))
conn.close()
